### ISSUE TITLE
App nextcloud non si connette

### ISSUE THREAD

**Issue by Giovannitomasi** [nextcloud/android#3238]
App nextcloud per android rimanda sempre al login dopo aver inserito login e pwd non si connette al server versione 14 app ultimo rilascio

**Comment by Giovannitomasi** [comment]
Accesso failed errore 401

**Comment by Giovannitomasi** [comment]
Errore 401 accesso failed e riporta al login

**Comment by tobiasKaminsky** [comment]
Can you create us a test account and send the credentials to tobias at nextcloud dot com with a reference to this issue?

**Comment by tobiasKaminsky** [comment]
The problem is that https://server/cloud/remote.php/webdav/ requires authentication, which is ok, but I cannot log in.

@nextcloud/server-triage do you have an idea how this could be checked during setup?
This happens quite often…

**Comment by tobiasKaminsky** [comment]
I wrote an email to both of you.

**Comment by labattazza** [comment]
Hi! I am the server administrator.
Thanks to Tobias that verified what is probably the cause of the issue.
There is anything I can check in the installation, to help discover what was wrong?
Or the other solution is to re-install,  after saving all the accounts I created meantime?

Could it depend on the slowness of the server causing some timeout ?
I also noticed that since the first installation the system complains about fault integrity of some files
